docker-compose.yaml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+  wsimod:
+    build: .
+    env_file:
+      - .env
```

Comment:
I believe this is redundant because docker compose uses `.env` by default

Reply:
Apparently it is not used automatically, no.

Reply:
Oh yes true, it uses it automatically for setting environment variables within the container, but not the way you're using them here